v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+# Group Owner
+
+## Owner [/owner]
+Access needs a __private key__.
+
+### List parking spaces [GET /owner/parking-spaces]
+Returns all parking spaces that are owned by the user given by the used private token for the request.
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Parking space)
+
+### Retrieve a Parking space [GET /owner/parking-spaces/{parking_space_id}]
+Retrieves the details of one singe parking space.
+
++ Parameters
+    + parking_space_id: `005c4826-4e28-11e3-a675-d43d7eece53d` (string, required) - The id or code of the parking space
```

Comment:
can you make this say only id instead of id or code? even when the source code supports using the ps code, I wouldn't mention that

Reply:
Done